src/config.rs

```diff
@@ -44,18 +44,18 @@ impl Config {
 		}
 	}
 
-	pub fn get_config(path: &PathBuf) -> ConfigFile {
+	pub fn get(path: &PathBuf) -> ConfigFile {
 		let result = std::fs::read_to_string(path);
 		match result {
 			Ok(content) => {
 				// TODO: Invalid structure of configuration file
 				serde_yaml::from_str(&content).unwrap()
 			}
-			Err(_) => Config::default_config(),
+			Err(_) => Config::default(),
 		}
 	}
 
-	pub fn default_config() -> ConfigFile {
+	pub fn default() -> ConfigFile {
```

Comment:
Implement https://doc.rust-lang.org/std/default/trait.Default.html#how-can-i-implement-default

Then you can just do things like:
```
Encryption {
  config: Default::default(),
}
```
```

Reply:
Wouldn't that only really be useful in tests? I don't see other places I really use the default like that.

Reply:
You can also do:
```rust
let config: Config = Default::default();
```

Which I feel like isn't actually as useful as when the type annotation already must exist (like in a struct or function param). I just really like `Default`, it's a cool thing that traits let you do, but yeah probably not super applicable here.